src/components/SearchBar.tsx

```diff
@@ -383,7 +383,7 @@ export function SearchBar({
     || (!isVertical && filteredRecentSearches?.length) || entityPreviews);
   const screenReaderText = getScreenReaderText(
     autocompleteResponse?.results.length,
-    filteredRecentSearches?.length,
+    isVertical ? 0 : filteredRecentSearches?.length,
     entityPreviewsCount
```

Comment:
Do we similarly need to guard the `entityPreviewsCount` as well? That doesn't appear on vertical searches iirc.

Reply:
We don't need a guard for `entityPreviewsCount` because there is no `renderEntityPreviews` function provided to the `SearchBar` unless VA is enabled (i.e. only on the universal page), so no `entityPreviews` are generated and the count will be 0 on vertical pages.